Art/Source/3D/Sequences/compose.py

```
@@ -0,0 +1,404 @@
+        for i, proc in enumerate(self.procedures):
+            print(f"  [{i+1}/{len(self.procedures)}] {proc.DESCRIPTION} @ {current_time:.1f}s")
+
+            # Animate this procedure
+            end_time = proc.animate(self.context, current_time)
+
+            # Add transition time before next procedure
+            if i < len(self.procedures) - 1:
+                # Animate camera transition
+                if camera:
+                    next_proc = self.procedures[i + 1]
+                    transition_start = seconds_to_frame(end_time)
```

Ruff (0.14.7)

Local variable `transition_start` is assigned to but never used

Remove assignment to unused variable `transition_start`

(F841)